src/main/java/com/sonyericsson/jenkins/plugins/bfa/statistics/StatisticsLogger.java

```diff
@@ -126,7 +126,7 @@ public void run() {
             Node node = build.getBuiltOn();
             String nodeName = node.getNodeName();
             int timeZoneOffset = TimeZone.getDefault().getRawOffset();
-            String master = "";
+            String master;
```

Comment:
Well at the same time, my IDE usually warns against such things with `Warning: var "master" may not have been initialized`. But it is far often ignored is my guess.